backend/benefit/calculator/models.py

```diff
@@ -345,6 +345,10 @@ class PreviousBenefit(UUIDModel, TimeStampedModel):
 
     history = HistoricalRecords(table_name="bf_calculator_previousbenefit_history")
 
+    @property
```

Comment:
This appears quite a lots in different models. Would it be better to create a mixin?

Reply:
Good idea!